scripts/test-runner.js

```diff
@@ -671,7 +671,6 @@ function run(args) {
         const isAdditionalFunctionTest = test.file.includes("__optimize");
         const isPureFunctionTest = test.name.includes("pure-functions");
         const isCaptureTest = test.name.includes("Closure") || test.name.includes("Capture");
-        const isSimpleClosureTest = test.file.includes("// simple closures");
```

Comment:
I'm none too sure what is going on here. At the very least this annotation should be removed from the test case that has it.

Reply:
Simple closures were removed a while back, I couldn't see why we had it in the codebase still, it confused me at first.